piwik/README.md

```diff
@@ -0,0 +1,31 @@
+# Piwik
+
+In a docker-compose.yml:
+```yml
+version: '3'
+services:
+  piwik:
+    image: quay.io/continuouspipe/piwik-php7.1-apache:latest
```

Comment:
we use `:stable` in the documentation, as we take cuts from `:latest` as stable tags when we're happy with them.